.github/workflows/codeql.yml

```diff
@@ -1,6 +1,11 @@
 name: "CodeQL"
 
 on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
```

Comment:
Why? To my knowledge the push and pull_request events are completely separate/unrelated.

Reply:
idk, that's what the codeql default config had. should i remove it?

Reply:
Up to you, I just find the comment confusing and misleading. The pull_request event will run on PR openings/updates and the push event will run when the PR is merged, and these are unrelated events. It should be possible to use only the push event or the pull_request event without the other.

Reply:
Now that I think about it this doesn't really make much difference, this is an unnecessary nitpick. Disregard it. PR looks good to go.